MapboxCoreNavigation/Constants.swift

```diff
@@ -62,12 +62,26 @@ public var MaxSecondsSpentTravelingAwayFromStartOfRoute: TimeInterval = 3
 
 
 /**
- Distance in meters for the minimum length of a step for giving a `medium` alert.
+ Distance in meters for the minimum length of a step for giving a `medium` alert while using the `MBDirectionsProfileIdentifier` `.automobileAvoidingTraffic` or `.automobile`
```

Comment:
The usual way to write this would be `MBDirectionsProfileIdentifierAutomobile` and `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`, since they’re defined as Objective-C constants.